### PR TITLE
Hello Microverse Pull Request

### PR DESCRIPTION
**Pull request by sahlminkok**
Add HTML, CSS, and Linters files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,213 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is copyright of [Sahalu Aminu]
+
+This project is [MIT](LICENSE.md) licensed.
```
Comment (ABDULALI3468) [inline README.md:211]: Currently, when a user clicks on the LICENSE link it results in a 404 ERROR. The reason is that you have used the wrong file path and name. Please consider reviewing it as it has a bad effect on user experience.

## Solution:--

Consider replacing the above line with the following and verify by clicking on it does it opens your LICENSE file

```
This project is [MIT](./LICENSE) licensed.
```

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @sahlminkok , 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**


**1-** Please consider always using the name of your feature branch as a way to highlight what you did in it (e.g. 'created_sidebar' or 'added_comments_section')  In this case, your branch name is `feature` which is not professional.

> Consider checking out from your current branch to a new branch and for this case, you can name it `feature-1/setup-basic-repo-configurations` or something else that describes what you did in the branch. Remember in the new PR which you will open for another review add the link of this PR in that PR's Description.

**2-** Please consider adding a professional description of your PR which describes what you have done in the Pull Request (PR). In this case, it's not describing it clearly.

>   pull_request_desc



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sahlminkok) [reply README.md:211]: @ABDULALI3468 How can I rename the branch without pulling another request

Comment (ABDULALI3468) [reply README.md:211]: @sahlminkok You need to check out from this branch into another branch on your computer by naming it in accordance with the above rules. Then you can commit the changes and push it to GitHub and open a new Pull Request after verifying if everything works fine.

**Pull request merged**